chromium_src/chrome/browser/ui/browser.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "brave/browser/ui/brave_browser_command_controller.h"
 #include "brave/browser/ui/brave_browser_content_setting_bubble_model_delegate.h"
+#include "brave/browser/ui/tabs/brave_tab_strip_model.h"
```

Comment:
why is this added here?

Reply:
I believe this is where you want to put it https://github.com/brave/brave-core/pull/4671/files#diff-3a289b88edecd27ffa752266d52b41aaR8

Reply:
I think what you want to do here is to create a chromium_src override for `chrome/browser/ui/browser.cc` and put the include there. That's the standard method for adding an include without patching

Reply:
oops, I looked at this wrong :)